### PR TITLE
BL-1294 alma holds that don't have expirations

### PR DESCRIPTION
**Pull request by quoideneuf**

### CODE REVIEW

**Comment by quoideneuf** [comment]
For the interim release: 
https://github.com/tulibraries/tul_cob/pull/new/v1.3.3-cherry-pic

File: spec/controllers/users_controller_spec.rb [hunk 1]
```diff
@@ -32,10 +32,38 @@
       end
 
       describe "GET #holds" do
+        render_views
+
         xit "returns http success" do
```
Comment (sensei100) [inline spec/controllers/users_controller_spec.rb:37]: @quoideneuf @dkinzer @bibliotechy I know this isn't part of these code changes, but in looking at this file I see we are skipping a number of tests.  It looks like that code was last changed 4 years ago.  Should we look at using these tests or remove them if they are not actually needed?

Comment (dkinzer) [reply spec/controllers/users_controller_spec.rb:37]: @sensei100 Yeah, I think that might be one of those default tests that gets added by spec template?  I'm OK with any of those getting removed.

Comment (quoideneuf) [reply spec/controllers/users_controller_spec.rb:37]: @dkinzer @sensei100 I can make a pass through these while I'm working on other error handling issues

**Pull request merged**